tests/HelperTest.php

```diff
@@ -88,17 +88,38 @@ public function testClear()
         $this->assertEmpty($_SESSION);
     }
 
+    /**
+     * @runInSeparateProcess
+     */
     public function testId()
     {
         $helper = new Helper();
 
         $this->assertSame(session_id(), $helper::id());
-        $this->markTestIncomplete('Please implement check for "::id(true)"');
+        $this->assertNotSame(session_id(), $sessionId = $helper::id(true));
+        $this->assertSame(session_id(), $sessionId);
     }
 
+    /**
+     * @runInSeparateProcess
+     */
     public function testDestroy()
     {
-        $this->markTestIncomplete('Please implement check for "::destroy"');
+        $helper = new Helper();
+
+        $_SESSION = ['a' => 'A', 'b' => 'B', 'c' => 'C'];
+
+        $this->assertNotEmpty($helper::id());
+        $this->assertNotEmpty(session_id());
+        $this->assertNotEmpty($_SESSION);
+
+        $helper::destroy();
+
+        $this->assertEmpty($helper::id());
+        $this->assertEmpty(session_id());
+        $this->assertEmpty($_SESSION);
+
+        $this->markTestIncomplete('Please finish test for "::destroy".');
```

Comment:
It's done:
- `session_unset` is covered in `assertEmpty($_SESSION)`.
- `session_destroy` is covered by `assertEmpty(session_id())`.
- `session_write_close` is not testable, the behavior varies between PHP installations.